livre/uvf/CMakeLists.txt

```diff
@@ -8,21 +8,9 @@ if(NOT TUVOK_FOUND)
   return()
 endif()
 
-set( LIVRE_UVF_HEADERS "UVFDataSource.h" )
-set( LIVRE_UVF_SOURCES "UVFDataSource.cpp" )
+set( LIVREUVF_HEADERS "UVFDataSource.h" )
+set( LIVREUVF_SOURCES "UVFDataSource.cpp" )
+set( LIVREUVF_LINK_LIBRARIES PRIVATE LivreCore Tuvok )
+set( LIVREUVF_INCLUDE_NAME livre/uvf)
 
-set( LIVRE_UVF_COMPILE_FLAGS "-std=c++0x" )
-set( LIVRE_UVF_TARGET_LIBRARIES LivreCore ${TUVOK_LIBRARY} )
-
-set( LIVRE_UVF_INCLUDES ${TUVOK_INCLUDE_DIR} )
-
-include_directories( ${LIVRE_UVF_INCLUDES} )
-add_library( LivreUVF SHARED ${LIVRE_UVF_SOURCES} )
-target_link_libraries( LivreUVF PRIVATE ${LIVRE_UVF_TARGET_LIBRARIES} )
-set_target_properties( LivreUVF PROPERTIES VERSION ${VERSION} SOVERSION ${VERSION_ABI} )
-set_target_properties( LivreUVF PROPERTIES COMPILE_FLAGS "${LIVRE_UVF_COMPILE_FLAGS}" )
-if(NOT APPLE)
-  set_target_properties( LivreUVF PROPERTIES LINK_FLAGS "-Wl,--no-undefined" )
-endif()
-install( TARGETS LivreUVF LIBRARY DESTINATION lib )
-install_files(include/livre/UVF FILES ${LIVRE_UVF_HEADERS} )
+common_library( LivreUVF ${LIVREUVF_HEADERS} ${LIVRE_UVF_SOURCES} )
```

Comment:
set_target_properties( LivreUVF PROPERTIES LINK_FLAGS "-Wl,--no-undefined" )		

This enforces the library to have no unidentified symbols ? Is it covered by the CMake subtree ?